config/routes.rb

```diff
@@ -2,5 +2,9 @@
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
   # Defines the root path route ("/")
-  # root "articles#index"
+  root "users#index"
+  # Define route for users in association with posts
+  resources :users do
+    resources :posts
+  end
```

Comment:
- [ ] Nice job but as part of the requirements, you have to make sure that you added code to handle only the 4 required URLs. i recommend using the `:only` key word